src/feature/payment/step/pick-recipients/recipient-search.jsx

```diff
@@ -1,17 +1,38 @@
-import React, { useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { Box } from '@material-ui/core';
+import React, {useEffect} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import {Box} from '@material-ui/core';
 import {
     updateSearchPage,
     updateSearchValue,
+    updateStep,
     updateSuggestionLength,
     updateSuggestions,
 } from '../../../../data/redux/actions/payment';
-import { GROUP, SEARCH_PAGE_ROWS, SEARCH_PAGE_START } from '../../constants';
+import {GROUP, SEARCH_PAGE_ROWS, SEARCH_PAGE_START, STEP_PICK_PRODUCTS} from '../../constants';
 import RecipientSuggestItem from './recipient-suggest-item';
 import SearchField from '../../../../common/search-field';
+import Button from "@material-ui/core/Button";
+import {makeStyles} from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Use Boxes if you can for layout instead of setting style with `makeStyles` and `className`.

Reply:
Fixed with Box, but couldnt get margin between buttons and had to still use styles.